hera_sim/visibilities/vis_cpu.py

```diff
@@ -82,7 +114,7 @@ def validate(self):
         super(VisCPU, self).validate()
 
         # This one in particular requires that every baseline is used!
-        N = len(self.uvdata.antenna_numbers)
+        N = len(self.uvdata.get_ants())
```

Comment:
get_ants() and ant_list produce antennas with data. antenna_numbers gets all antennas. This line has to use antenna_numbers otherwise the test that follows it won't work. Changed back to antenna_numbers.

Reply:
OK.